kartograf/bogon.py

```diff
@@ -133,19 +136,12 @@ def is_bogon_pfx(prefix):
     - https://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry.xhtml
     - https://bgpfilterguide.nlnog.net/guides/bogon_prefixes/
     """
-    network = ipaddress.ip_network(prefix)
-    version = network.version
-
-    if version == 4:
-        for ipv4_range in SPECIAL_IPV4:
-            if network.subnet_of(ipaddress.ip_network(ipv4_range)):
-                return True
-    elif version == 6:
-        for ipv6_range in SPECIAL_IPV6:
-            if network.subnet_of(ipaddress.ip_network(ipv6_range)):
-                return True
-
-    return False
+    try:
```

Comment:
Hm, this silences the error. I don't think that's what we want here. Generally such an error shouldn't happen here of course but if it does something has been going really wrong and then this function shouldn't pretend that the network is actually fine and not even a bogon. We would probably run into issue later and have a harder time debugging the later that happens.